packages/mco/components/mco-dashboard/data-policy/cluster-app-card/cluster-app-card.scss

```
@@ -20,6 +20,10 @@
     &__dropdown--padding {
         padding-right: var(--pf-global--spacer--xs);
     }
+    &__dropdown--maxHeight {
```

not here (`--maxHeight`) because u are also setting `overflow`, nit just `maxHeight`...
```suggestion
    &__dropdownHeight {
```